### DISCUSSION TITLE
Few questions

### DISCUSSION THREAD

**Discussion by sm18lr88** [AI4Finance-Foundation/FinGPT#69] [General]
Hi. I've been eager to learn about your project.

1. In learning about your project I came across a "FinGPT" website called Fintips.io that does sentiment analysis. I don't think they're related, but just asking.

2. In learning about PEFTs, I came across AdaLoRa, which seems to be cheaper and better than LoRa in many ways. Have you heard about it?

3. I saw a benchmark for v3.3 (OpenAi). May I know what that entails? I haven't found any details.

4. Last question: do you think companies will start using AI to create comments designed to manipulate sentiment in social media as more people use it for investment purposes?

Thanks!

**Comment by BruceYanghy** [comment]
1. I noticed numerous domain names associated with FinGPT, but none of them are affiliated with us. Thank you.

**Reply by drizzy765** [reply]
hmmm
